conf/ref_databases.config

```diff
@@ -382,4 +382,74 @@ params {
             dbversion = "UNITE-alleuk v8.2 (https://dx.doi.org/10.15156/BIO/786376)"
         }
     }
+    // Kraken2 reference databases
+    kraken2_ref_databases {
+        // Corresponding S3 URLs can be obtained by removing https://genome-idx.s3.amazonaws.com from the beginning of the URLs linked to above and replacing with s3://genome-idx.
+        'silva' {
+            title = "Kraken2 pre-formatted SILVA - Version 138"
+            file = [ "https://genome-idx.s3.amazonaws.com/kraken/16S_Silva138_20200326.tgz" ]
```

Comment:
are the DBs versioned or can the data behind the link change at any time?

Reply:
The key 'silva' has always the latest version, currently its identical with 'silva=138', 'silva=132' is an older version. The files chosen when using 'silva' can indeed change from release to release.

Reply:
so no reproducibility 😢 ?

Reply:
Given the same pipeline version, 'silva' will always be the same version. Across pipeline versions, 'silva' will be always the latest version, i.e. differ. By using 'silva=138' the database will be fixed across pipeline versions.
I would call that reproducible.

Reply:
ah ok, then all good, forget my comment :) I misunderstood you earlier, I thought you meant it updates to latest on every run.